### DISCUSSION TITLE
Backward compatibility

### DISCUSSION THREAD

**Discussion by glupol** [microsoft/onnxruntime#14656] [Other Q&A]
Hi I need to install and use onnx on an old platform that does not support new c/c++ standards.
Is there an onnx[runtime] c/c++ version that does not relies on the new standards?
The available compilers are vs 2008 and gcc-4.6
Maybe some previous version of onnx ?

Best,
Greg.

**Comment by justinchuby** [comment]
This is likely too old of a version onnx never supported
